digid_eherkenning/saml2/eherkenning.py

```diff
@@ -32,6 +35,214 @@
 xml_nl_lang = {"{http://www.w3.org/XML/1998/namespace}lang": "nl"}
 
 
+def generate_dienst_catalogus_metadata(eherkenning_config):
+
+    key_file = eherkenning_config.certificate.private_key.path
+    cert_file = eherkenning_config.certificate.public_certificate.path
+
+    settings = {
+        "key_file": key_file,
+        "cert_file": cert_file,
+        "base_url": eherkenning_config.base_url,
+        "entity_id": eherkenning_config.entity_id,
+        "oin": eherkenning_config.oin,
+        "organisation_name": eherkenning_config.organization_name,
+        "services": [
+            {
+                "attribute_consuming_service_index": eherkenning_config.eh_attribute_consuming_service_index,
+                "service_loa": eherkenning_config.loa,
+                "service_uuid": str(uuid4()),
```

Comment:
does it make sense that this UUID is different every time metadata is generated?